airgun/views/contentview_new.py

```diff
@@ -61,3 +69,99 @@ def is_displayed(self):
     def after_fill(self, value):
         """Ensure 'Create content view' button is enabled after filling out the required fields"""
         self.submit.wait_displayed()
+
+
+class NewContentViewEditView(BaseLoggedInView):
+    """Inside a cv editable view that contains 'Details', 'Versions', 'Repo', 'Filters', and 'History'"""
+    breadcrumb = BreadCrumb('cv-breadcrumb')
+    title = Text('//h1[@data-ouia-component-id="cv-details-header-name"]')
+    kebab = Dropdown(
+        "//div[contains(@data-ouia-component-id, 'OUIA-Generated-Dropdown') or "
+        "@data-ouia-component-type='PF4/Dropdown']"
+    )
+    publish = Button('cv-details-publish-button')
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and len(self.breadcrumb.locations) <= 3
+                and self.breadcrumb.locations[0] == 'Content Views'
+                and self.breadcrumb.read() != 'New Content View'
+        )
+
+    @View.nested
+    class details(SatTab):
+        name = EditableEntry(name='Name')
+        label = ReadOnlyEntry(name='Label')
+        description = EditableEntry(name='Description')
+        solve_dependencies = Switch(id='Solve Dependencies')
+
+    @View.nested
+    class versions(SatTab):
+        searchbox = PF4Search
+        table = PatternflyTable(
+            component_id='content-view-versions-table',
+            column_widgets={
+                0: Checkbox(locator='.//input[@type="checkbox"]'),
+                'Version': Text('./a'),
+                'Packages': Text('.//a'),
+                'Errata': Text('.//a'),
+            },
+        )
+
+        def search(self, version_name):
+            """Searches for content view version.
+
+            Searchbox can't search by version name, only by id, that's why in
+            case version name was passed, it's transformed into recognizable
+            value before filling, for example::
+
+                'Version 1.0' -> 'version = 1'
+            """
+            search_phrase = version_name
+            if version_name.startswith('V') and '.' in version_name:
+                search_phrase = f'version = {version_name.split()[1].split(".")[0]}'
+            self.searchbox.search(search_phrase)
+            return self.table.read()
+
+
+    @View.nested
+    class repos(SatTab):
+        pass
+
+    @View.nested
+    class filters(SatTab):
+        pass
+
+
+class NewContentViewPublishView(BaseLoggedInView):
```

Comment:
This needs to use `baseModal` instead of the `BasedLoggedInView`.  The modal for publish uses both `wizard` and modal so it can be confusing to write something that makes sense here.